# NtcA_bioprotocol_tutorial
samples for ChIP-seq data analysis described in "ChIP-seq Experiment and Data Analysis in the Cyanobacterium Synechocystis sp. PCC 6803" bio-protocols
Samples for ChIP-seq data analysis described in the bioinformatic tutorial "ChIP-seq Experiment and Data Analysis in the Cyanobacterium Synechocystis sp. PCC 6803"

Authors: Joaqu√≠n Giner-Lamia, Miguel Hernandez-Prieto, M.E. Futschik

Journal: Bio-protocols

The data deposited in this project comes from a series of ChIP-seq experiments to analyse the reguon of NtcA in conditions of low and high avilability of nitrogen. The analysis of these data can be found at:

Joaquin Giner-Lamia et al., Identification of the direct regulon of NtcA during early acclimation to nitrogen starvation in the cyanobacterium Synechocystis sp. PCC 6803. 2017. Nucleic Acids Res.

FastQ files presents in this repository has been modify to minimize the computational time consuming for the bioinformatic analysis during the protocols. Thus files presents in this repository contains only 1% of total reads presents in the original ChIP-seq files.

The original fastaQ files are available at Gene Expression Omnibus(GEO) with the accession number: GSE97291.

**Files description:**

**N_input.fastq:** input sample from Synechocystis cells grown 4 hours in absence of nitrogen.

**N_ChIP.fastq:** ChIP-seq sample from Synechocystis cells grown 4 hours in absence of nitrogen.

**NH4_input.fastq:** input sample from Synechocystis cells grown 4 hours in presence of ammonium.

**NH4_ChIP.fastq:** ChIP-seq sample from Synechocystis cells grown 4 hours in presence of ammonium.
