#This R script has been created for the ChIP-seq data analysis tutorial publicated in:
# Joaqun Giner-Lamia, Miguel Hern√°ndez-Prieto, M.E. Futschik. "ChIP-seq Experiment and Data Analysis in the Cyanobacterium Synechocystis sp. PCC 6803". 2018. bio-protocols
# For more information visit the wiki Github page: https://github.com/ginerorama/NtcA-bio-protocols-tutorial/wiki


#Install GenomicFeatures and ChIPseeke
source("https://bioconductor.org/biocLite.R")
biocLite("GenomicFeatures")
biocLite("ChIPseeker")


#Creating a TxDb using makeTranscriptDbFromGFF function from GenomicFeatures
library(GenomicFeatures) #Load GenomicFeatures Package
setwd()#path to tutorial files in your computer
txdb <- makeTxDbFromGFF('NC_000911.1.gff', format='gff')
genes <- genes(txdb)


#peak annotation using ChIPseeker.
library(ChIPseeker)#Load ChIPseeker Package
peakfile= 'NtcA_N_peaks.bed' #bed file generated by MACS
peakAnno <- annotatePeak(peakfile, tssRegion=c(-300,300), TxDb=txdb) # annotate peaks
write.table(peakAnno,file = 'N_annotated_peaks.txt', sep='\t') # save theannotate peaks in .txt file 


#some nice graphic graphic to visualize your ChIP-seq data try it..!!!
upsetplot(peakAnno, vennpie=TRUE)
vennpie(peakAnno)
plotAnnoPie(peakAnno)


#########################################################